hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/conf/Configuration.java

```diff
@@ -573,7 +573,7 @@ public static void addDeprecations(DeprecationDelta[] deltas) {
    * It does not override any existing entries in the deprecation map.
    * This is to be used only by the developers in order to add deprecation of
    * keys, and attempts to call this method after loading resources once,
-   * would lead to <tt>UnsupportedOperationException</tt>
+   * would lead to <code>UnsupportedOperationException</code>
```

Comment:
Thanks for creating this PR @ayushtkn. Would you mind creating this tag change as a separate PR? I believe it's not relevant.

Reply:
the javadoc build was failing, with jdk-11, some html5 becomes default and the tags that I removed/changed were removed from that...
Try reverting one line and try to build javaDoc it will fail

Reply:
Gotcha. Thanks for clarifying.